recipes/ms/meta.yaml

```diff
@@ -0,0 +1,22 @@
+package:
+  name: ms
+  # No version numbers are supplied, so we use the last updated date in the 
+  # header of ms.c as the version number.
+  version: "2014_03_04"
+
+build:
+  number: 0
+
+source:
+  fn: ms.tar.gz
+  url: https://webshare.uchicago.edu/users/rhudson1/Public/ms.folder/ms.tar.gz
+  md5: ef693dcc424fd5293b0bea4c68953700
+
+test:
+  commands:
+    - ms 2 1 -T > /dev/null
+
+about:
+  home: http://home.uchicago.edu/rhudson1/source/mksamples.html
```

Comment:
Ok. Maybe we can add ``license: Unknown``? Then, it is clear at least that we checked and did not just forget to add the information.